Utils/Win32ErrorCodeException.h

```diff
@@ -1,6 +1,6 @@
 #pragma once
-
 #include <string>
+#include <Windows.h>
```

Comment:
You don't need this header here, only in the source file.
Maybe there is problem with you setup (Visual Studio).
Try to remove it and if it isn't solve the problem talk with me.